_posts/2024-03-26-Announcing-Data-Prepper-2.7.0.md

```diff
@@ -99,13 +99,18 @@ To support [secret rotation](https://docs.aws.amazon.com/secretsmanager/latest/u
 
 For more information, please see the [`aws` extension](https://opensearch.org/docs/latest/data-prepper/managing-data-prepper/configuring-data-prepper/#aws-extension-plugins) documentation.
 
+This feature is experimental while we work to add more support for refreshing certain fields and dynamically updating those fields.
+In particular, the `opensearch` sink and the `kafka` plugins do not automatically refresh secrets.
```

Comment:
I'm assuming "plugins" is intentionally plural.

Reply:
Yes, there are two `kafka` plugins.